### DISCUSSION TITLE
Is CryptoSwift compatible with ARM PSA?

### DISCUSSION THREAD

**Discussion by georgehuardtechserv** [krzyzanowskim/CryptoSwift#993] [Q&A]
We are trying some encryption and decryption via Bluetooth but we are getting different results. What we are doing is: I am creating a random text to encrypt and then sending it over. I then receive the encrypted message and IV and decrypt it, but the result does not match the original text I sent. 

We have run tests locally on each side and the encryption/decryption works fine, but it is once we start sending data to each other that the values change.

**Comment by krzyzanowskim** [comment]
I;m not sure how I could help you with that
